### DISCUSSION TITLE
Is possible to disabled some unnecessary calls?

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-js#150] [Q&A] [answered]
Hi, Im using js SDK and in the documentation says that is possible to disable the tracking of generic metrics, but when i use this property there are still multiple calls to this endopoint: https://events.mercadopago.com/v2/metric, I dont know if this a bug, or even with property  `trackingDisabled` added this calls needs to be called for some reason?, thanks 

<img width="714" alt="Captura de Pantalla 2023-04-14 a la(s) 09 58 06 a  m" src="https://user-images.githubusercontent.com/73070103/232050073-5158e0e7-a6e8-4808-ae9a-92e742b9b714.png">

**Comment by AdaltonLeite** [comment, marked as answer]
Hi @joxpulp ,

Currently our SDK allows you to disable features metrics. The one you saw in network `/metrics` is a basic metric for the [health](https://status.mercadopago.com/) of the application and therefore cannot be disabled.

I hope that helps, thank you!

**Reply by joxpulp** [reply]
Thanks 🙌🏽
